x/harvest/keeper/borrow.go

```diff
@@ -19,11 +19,20 @@ func (k Keeper) Borrow(ctx sdk.Context, borrower sdk.AccAddress, coins sdk.Coins
 		}
```

Comment:
Just realized this, but we should check that the the money market exists in params before calling `k.SetBorrowIndex`, as a user could input a coin that doesn't exist and we would still store a borrow index for it.

Reply:
Good catch, updated in https://github.com/Kava-Labs/kava/pull/742/commits/75c2fcd81848211ff3f15d17f3b7437abc50f222